cdn/signedurls/signurl.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command signedurls contains helper function to sign URLs.
```

Comment:
sign URLs for what? can you make this more descriptive?

Reply:
Also, if it's a helper function, then it's kinda weird to have it as a command.

I think just describe what the program does.